examples/sine_advanced.rs

```diff
@@ -101,7 +120,7 @@ fn main() -> Result<(), coreaudio::Error> {
     println!("setting hardware (physical) format");
     let hw_stream_format = StreamFormat {
         sample_rate: SAMPLE_RATE,
-        sample_format: SampleFormat::I24,
+        sample_format: SampleFormat::I16,
```

Comment:
why was this changed?

Reply:
I needed to change it to run on the headphone output of my old MacBook. Then I decided to not change it back, thinking that I16 probably is a better choice. I think this should allow the example to run unmodified on more hardware.

Reply:
Ah ok, sounds good.